ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,23 +159,40 @@ objects:
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
         function fips_check() {
-          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
-          while IFS= read -r i; do
-            oc -n default --insecure-skip-tls-verify --request-timeout=60s debug --image centos:7 node/$i -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
+          get_nodes=$(oc --request-timeout=20s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}')
+          nodes=( $get_nodes )
+          # bash doesn't handle '.' in array elements easily
+          for i in {0..5}; do
+            attempt=0
+            while true; do
+                out=$(oc -n default debug node/"${nodes[i]}" -- cat /proc/sys/crypto/fips_enabled || true)
+                if [[ ! -z "{$out}" ]]; then
```

Comment:
This looks wrong

Reply:
it is, fixed : (

Reply:
I know you have better things to do w/ your time than catch my bash errors - i cp'd wrong from my local script where i've been testing the retry : ( 
Now i can't get oc debug node to fail so that's good, but if so i would've caught this, thanks, though.